src/core/lib/surface/lame_client.cc

```diff
@@ -79,6 +80,7 @@ static void fill_metadata(grpc_exec_ctx *exec_ctx, grpc_call_element *elem,
 static void lame_start_transport_stream_op_batch(
     grpc_exec_ctx *exec_ctx, grpc_call_element *elem,
     grpc_transport_stream_op_batch *op) {
+  CallData *calld = static_cast<CallData *>(elem->call_data);
```

Comment:
`static_cast` is valid but gives me the creeps in a situation like this when converting from `void *` to a real pointer type. This feels like it should be a `reinterpret_cast` morally, but I know both are legal in this case. I suppose the style guide suggests using `static_cast` because it's weaker, but it still feels icky.